app/views/eventRegistrations.html

```diff
@@ -30,31 +30,51 @@ <h2 class="page-title">Registrations</h2>
           <option value="over">Overpaid</option>
         </select>
       </div>
-      <div class="col-md-2 col-sm-7 stacked-spacing-col-sm">
+      <div class="col-md-2 col-sm-6 stacked-spacing-col-sm">
         <label>Registrant type:</label>
         <select ng-model="queryParameters.filterRegType" ng-options="t.id as t.name for t in visibleFilterRegistrantTypes" class="form-control"></select>
       </div>
-      <div class="col-md-4 col-sm-5 text-right">
+      <div class="col-md-5 col-sm-6 text-right stacked-spacing-col-sm">
```

Comment:
I'm not sure the buttons need space below them (`stacked-spacing-col-sm`) since they are the last items in the well.